api/v1/endpoints/get-identity-map.md

```diff
@@ -18,13 +18,15 @@ Retrieve the UID2 and salt bucket ID for an email address or an email address ha
 
 ###  Query Parameters
 
-* Only one of the following two query parameters is required. 
+* Only one of the following four query parameters is required. 
```

Comment:
Should this be "Exactly on of ... "?

Reply:
Reworded to a single important note: You must include only one of the following parameters.